<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JarvisUi</class>
 <widget class="QMainWindow" name="JarvisUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1608</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1611</width>
      <height>821</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dutch801 XBd BT</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="movie">
     <string>your_app_path/assets/interface.gif</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Activate">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>617</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color: rgb(44, 223, 202);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>730</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>color: rgb(0, 222, 222);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Terminate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_8">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>70</y>
      <width>256</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_9">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>240</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_10">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>450</y>
      <width>256</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius: none</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dutch801 Rm BT</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>color: rgb(0, 220, 220);</string>
    </property>
    <property name="text">
     <string>J.A.R.V.I.S</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1608</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
